E-Commerce/readme.md

```diff
@@ -46,16 +46,87 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+
+```sql
+SELECT * FROM products WHERE product_name ILIKE '%socks%';
```

Comment:
CORRECT (note that ILIKE is not case specific - although this is irrelvant with this data set)